src/NUnitConsole/nunit3-console/ResultReporter.cs

```diff
@@ -191,7 +191,7 @@ private void WriteErrorsFailuresAndWarnings(XmlNode resultNode)
                         else
                         {
                             var site = resultNode.GetAttribute("site");
-                            if (site == "SetUp" || site == "TearDown")
+                            if (site != "Parent" && site != "Child")
```

Comment:
Is it possible to protect this with a test?

Reply:
Actually, I think that this is wrong 😄 as the `site` is `null` for warnings, hence we will write one warning for every step down to the actual warning.

I - more or less - did the opposite change, so we did not have to introduce a null check (also to align the code with NUnitLite). See https://github.com/nunit/nunit-console/commit/d5822fec5af7633a4f1c25caf9585e8d8c3fcbc8#diff-16ed6eb93d79ef9b2ca6bd51950d69ba

Also see here for the discussion #326.